rpi-eeprom-update

```diff
@@ -400,6 +400,7 @@ Options:
       flag use the config in that file. This option only applies when a
       bootloader EEPROM update is needed; if the bootloader EEPROM is up-to-date
       then its config will not be changed.
+      Without this switch, the current bootloader config will be retained.
```

Comment:
See line 392? :wink:

Reply:
I think the preservation of the EEPROM config is already specified.
The "Use the default bootloader config" text could be removed because there is no default bootloader-config, just whatever the eeprom image contains.
aka -d - don't migrate EEPROM config